contracts/SponsorFaucet.sol

```diff
@@ -131,95 +97,80 @@ contract SponsorFaucet is Ownable, Pausable, ReentrancyGuard, WhitelistedRole {
 
     //set bounds for sponsorship
     function setBounds(
-        uint256 gasTotalLimit,
-        uint256 collateralTotalLimit,
-        uint256 gasBound,
-        uint256 collateralBound,
-        uint256 upperBound
+        bounds memory smallBounds
     ) public {
         _onlyWhitelistAdmin();
         //rule by internal contract
-        require(upperBound.mul(1000) <= gasBound, "upperBound too high");
-        gas_total_limit = gasTotalLimit;
-        collateral_total_limit = collateralTotalLimit;
-        gas_bound = gasBound;
-        collateral_bound = collateralBound;
-        upper_bound = upperBound;
+        require(smallBounds.upper_bound.mul(1000) <= smallBounds.gas_bound, "upperBound too high");
+        dapp_bounds[small] = smallBounds;
     }
 
     //set bounds for general contracts
     function setGeneralBounds(
-        uint256 gasTotalLimit,
-        uint256 collateralTotalLimit,
-        uint256 gasBound,
-        uint256 collateralBound,
-        uint256 upperBound
+        bounds memory largeBounds
```

Comment:
Why not change the function and its comment as well?

Reply:
commets for **public** methods added